src/contracts/Event/AfterSuggestionEvent.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+/**
+ * @copyright Copyright (C) Ibexa AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace Ibexa\Contracts\Search\Event;
+
+use Ibexa\Contracts\Core\Repository\Event\AfterEvent;
+use Ibexa\Contracts\Search\Model\Suggestion\SuggestionCollection;
+use Ibexa\Search\Model\SuggestionQuery;
+
+final class AfterSuggestionEvent extends AfterEvent
+{
+    private SuggestionQuery $query;
+
+    private SuggestionCollection $suggestionCollection;
+
+    public function __construct(SuggestionQuery $query, SuggestionCollection $suggestionCollection)
+    {
+        $this->query = $query;
+        $this->suggestionCollection = $suggestionCollection;
+    }
+
+    public function setQuery(SuggestionQuery $query): void
+    {
+        $this->query = $query;
```

Comment:
What's the point of allowing the query to be overwritten, when we already have the results?